etl/run_etl.R

```diff
@@ -0,0 +1,36 @@
+library(redcapcustodian)
+library(dotenv)
+library(callr)
+library(argparse)
+
+set_script_run_time()
+
+parser <- ArgumentParser()
+parser$add_argument("script_name", nargs=1, help="Script to be run")
```

Comment:
Could you allow for optional parameters? it's quite rare but we do pass an argument to one of out ETLs in the rcc.ctsit repo